site/_scss/blocks/_search-box.scss

```diff
@@ -103,12 +103,15 @@ search-box[active] {
 
 .search-box__snippet {
   color: var(--color-secondary-text);
+  word-break: break-all;
```

Comment:
You can remove this. Otherwise it causes words to wrap in strange ways like breaking at the last letter. The words seem to wrap correctly now with the margin that you added.